### PR TITLE
CRM-20313 Add index to civicrm_activity.status_id

### PR DESCRIPTION
**Pull request by JoeMurray**
UI_status_id

---

 * [CRM-20313: Add index to civicrm_activity.status_id](https://issues.civicrm.org/jira/browse/CRM-20313)

### CODE REVIEW

File: xml/schema/Activity/Activity.xml [hunk 1]
```diff
@@ -263,6 +263,11 @@
       <type>Select</type>
     </html>
   </field>
+  <index>
+    <name>UI_status_id</name>
```
Comment (JoeMurray) [inline xml/schema/Activity/Activity.xml]: Seems like this naming standard has changed and the current preferred default prefix is index_*

Please update.

**Comment by Edzelopez** [comment]
Closing in favour of https://github.com/civicrm/civicrm-core/pull/10056

**Comment by Edzelopez** [comment]
Please close in favour of https://github.com/civicrm/civicrm-core/pull/10056

**Comment by JoeMurray** [comment]
Closing in favour of https://github.com/civicrm/civicrm-core/pull/10056/files

**Pull request closed without merging**